player/loadfile.c

```diff
@@ -132,7 +132,10 @@ static void print_stream(struct MPContext *mpctx, struct track *t)
     if (t->title)
         APPEND(b, " '%s'", t->title);
     const char *codec = s ? s->codec->codec : NULL;
-    APPEND(b, " (%s)", codec ? codec : "<unknown>");
+    if (codec && s->codec->channels.num)
+        APPEND(b, " (%s, %s)", codec, mp_chmap_to_str(&s->codec->channels));
+    else
+        APPEND(b, " (%s)", codec ? codec : "<unknown>");
```

Comment:
Doesn't follow coding style (add `{`/`}` to if statements with else branch). Also, I think it'd be more logical to break the codec==NULL case a separate if branch as well.